hail/python/hail/plot/plots.py

```diff
@@ -1491,11 +1498,15 @@ def manhattan(pvals, locus=None, title=None, size=4, hover_fields=None, collect_
 
     pvals = -hail.log10(pvals)
 
+    if collect_all:
+        warnings.warn('manhattan: `collect_all` has been deprecated. Use `downsample` instead.')
+        downsample = False
```

Comment:
If a user specifies `manhattan(collect_all=False, downsample=False)` they should get an error. As written, I think that would not downsample (violating the `collect_all` setting). I think your best bet is to change the default argument for `collect_all` to `None` and then:
1. Verify only one of the two is not `None`.
2. Warn if `collect_all` is not `None`.

I know this adds a fair bit of code bloat, but it seems worthwhile if it saves users from getting confused by doubly-specified parameters.

Reply:
I had considered doing that, but questioned the benefit. I'll add that in now

Reply:
Seems to me we already have double specificity by the existance of `n_divisions`.